### ISSUE TITLE
Dragon ball z vs tag ( ARM64 git bug whit slow effect enabled)

### ISSUE THREAD

**Issue by brujo5** [hrydgard/ppsspp#7664]
This happend When slow effects are oN

![image](https://cloud.githubusercontent.com/assets/4172898/7018016/e7156c1c-dcbc-11e4-81d0-0d777a376da5.jpg)

But is still playable in non befered rendering
![image](https://cloud.githubusercontent.com/assets/4172898/7018025/06c4ca26-dcbd-11e4-8552-9d000a34ecfb.jpg)
Bufered rendering + slow effects OFF

**Comment by brujo5** [comment]
Also noticed that kingdom hearth birth sleep is forced to run at 60fps without cheats. 

Cause to run slow, is normal???

![image](https://cloud.githubusercontent.com/assets/4172898/7018456/e6526ae6-dcc1-11e4-838d-3ecf53635ac2.jpg)

**Comment by hrydgard** [comment]
That is odd. What hardware are you playing on here? You usually report iOS bugs right? I don't think anyone has enabled ARM64 builds on iOS yet. If you see "Experimental ARM64 build" in red letters on startup, you know you are running ARM64.

**Comment by brujo5** [comment]
My last working build until yesterday was V1.0.1-236 and no idea what broke slow effects in the most recent build. 1.0.1-343

thanks for the explanation About ARM64 GIT.

**Comment by brujo5** [comment]
@rock88 

Hey Man, any chance to add support ARM64 GIT for iPhones??

**Comment by hrydgard** [comment]
Regarding the 30/60 fps problem, have you tried enabling/disabling the multithreading option?

**Comment by brujo5** [comment]
@hrydgard 

Yes, for example here in old build 1.0.1-118 the game working perfect 30/30 fps

![image](https://cloud.githubusercontent.com/assets/4172898/7076387/3d267f50-dec6-11e4-8d0f-a567e5c7946f.jpg)


But still in the most recent build 1.0.1-381 the game is forced to 60fps
![image](https://cloud.githubusercontent.com/assets/4172898/7076408/6c08d1d8-dec6-11e4-89e2-c90d4edf3a84.jpg)

**Comment by unknownbrackets** [comment]
Might be related to settings.  Odd things can happen if you've modified the cpu clock or various other settings, and especially the timer hack.

For me, on Windows, the game runs at 30fps still and appears to be fine.  This isn't the Final Mix version.

-[Unknown]

**Comment by brujo5** [comment]
Nope, delete settings and still the same.

**Comment by brujo5** [comment]
@unknownbrackets 

Any idea if this changes 1.0.1-177 in depal shader broke dvz vs tag

**Comment by brujo5** [comment]
Here an better explanation..

http://es.tinypic.com/r/2zzhh5k/8

**Comment by unknownbrackets** [comment]
The biggest thing I can imagine changing from #7600 is failure cases, like if the shader was already failing to compile.  We'd get reports of that assuming you have reporting enabled in settings.

So you're saying that version doesn't work and the immediately previous revision, v1.0.1-174-g37344d1, does?

-[Unknown]

**Comment by brujo5** [comment]
Well, my backup (stable build) reach 1.0.1-169, working fine.
and have no more recent.

**Comment by brujo5** [comment]
Kingdom hearth birth sleep fps fixed in latest build, homeweber dbz vs tag still not fixed.

![image](https://cloud.githubusercontent.com/assets/4172898/8100632/c4f3fdf0-0fcb-11e5-8e15-4572f6f4a69b.jpg)

**Comment by brujo5** [comment]
@hrydgard 

how can enable the arm64 jit in android?

**Comment by hrydgard** [comment]
@brujo5 if your device has a 64-bit ARM CPU, the arm64 jit will be automatically used.

**Comment by brujo5** [comment]
@hrydgard 

thanks

i notice that in dbz tenkiachi vs tag still dont have the outline character 

tested in samsung galaxy s6

**Comment by unknownbrackets** [comment]
That's a GPU / GPU driver issue and has nothing to do with arm64 or not.  So that's a Mali-T760 MP8.

The effect depends on depal working correctly and also absdiff working correctly.  If they're off-by-one or rounding incorrectly, that's likely enough to make it draw incorrectly.

-[Unknown]

**Comment by brujo5** [comment]
whats the rute for place my custom shaders in android?

navigate to @root data/data/org.ppssppp.ppsspp/

cant find the shader folder

also replaced the shader folder inside the apk but cause install error.

help me, please

**Comment by unknownbrackets** [comment]
https://github.com/hrydgard/ppsspp/commit/74db6d4e266f2ebd1d7b61309eab934c21b60a4c

It's /PSP/shaders/.  You don't need to try mucking around inside the PSP's data, which I don't think will  ork (it's inside the APK) and would get overwritten on upgrade anyway.  That's why /PSP/shaders/ exists.

-[Unknown]

**Comment by brujo5** [comment]
That works thanks.

**Comment by unknownbrackets** [comment]
How do things look here today?

-[Unknown]

**Comment by brujo5** [comment]
i dont have an iphone anymore. but in android are perfect since long time.
